### PR TITLE
Sailor update

### PR DESCRIPTION
**Pull request by uaArsen**
**DoD**
**Development:**
- [x] use new logger
- [x] component should be dockerised, add: "buildType":"docker" line into component.json
- [x] sailor version is changed from 2.5.1 to 2.5.4
- [x] CHANGELOG.md is updated
**QA:**
- [ ] Smoke test is finished
- [ ] Regression testing is finished (optional)
- [ ] Component is deployed to the production stage (app.elastic.io) and/or another required stage

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ ## 1.1.0 (January 21, 2020)
```
Comment (denyshld) [inline CHANGELOG.md]: Version change required

Comment (kirill-levitskiy) [inline CHANGELOG.md]: The same version as below, please replace

File: package.json [hunk 2]
```diff
@@ -22,14 +22,15 @@
...
-    "co-mssql": "^1.3.0",
+    "bluebird": "3.4.6",
+    "co": "4.6.0",
+    "co-mssql": "1.3.0",
     "elasticio-node": "0.0.9",
-    "elasticio-sailor-nodejs": "^2.2.1",
-    "mssql": "^4.1.0",
-    "request": "^2.87.0",
-    "request-promise": "^4.2.2"
+    "elasticio-sailor-nodejs": "2.5.4",
```
Comment (kirill-levitskiy) [inline package.json]: I'd recommend to use the latest version of sailor (now it is `2.6.1`)

**Review by kirill-levitskiy — CHANGES_REQUESTED** [review]

**Review by kirill-levitskiy — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+ * Update sailor version to 2.5.4
```
Comment (stas-fomenko) [inline CHANGELOG.md]: should be 2.6.1

**Review by stas-fomenko — CHANGES_REQUESTED** [review]

**Pull request merged**